components/BlurhashImage.vue

```diff
@@ -56,14 +72,40 @@ export default Vue.extend({
       image: ''
     };
   },
+  computed: {
+    getImageHash(): string {
+      if (
+        this.imageType === ImageType.Backdrop &&
+        this.item?.BackdropImageTags?.[0] &&
+        this.item?.ImageBlurHashes?.Backdrop
+      ) {
+        return this.item?.ImageBlurHashes?.Backdrop?.[
+          this.item?.BackdropImageTags?.[0]
+        ];
+      } else if (
+        this.imageType === ImageType.Backdrop &&
+        this.item?.ImageTags?.Thumb &&
+        this.item.ImageBlurHashes?.Thumb?.[this.item?.ImageTags?.Thumb]
+      ) {
+        return this.item.ImageBlurHashes?.Thumb?.[this.item?.ImageTags?.Thumb];
+      }
+      if (
+        this.item?.ImageTags?.Primary &&
+        this.item.ImageBlurHashes?.Primary?.[this.item?.ImageTags?.Primary]
+      )
+        return this.item.ImageBlurHashes?.Primary?.[
+          this.item?.ImageTags?.Primary
+        ];
+      // It should never reach this since there are v-ifs above that shouldn't call
+      // getImagehash() if the relavent image tags are not there. But in the case that it does
+      // This hash is here to prevent errors.
+      return 'L7F$k?_*41GX^]KhTnJ8G?OXvz#;';
```

Comment:
Instead of doing this, we should probably check if the hash is a valid one, and disable blurhash if it isn't.

Reply:
I've introduced a check in `BlurhashCanvas.vue` to check if the hash is valid. If it fails, the canvas is hidden